lib/screen/home_screen.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:sochem/utils/constants.dart';
+import 'package:sochem/widgets/carousel.dart';
+import 'package:sochem/widgets/gridcards.dart';
+
+class HomeScreen extends StatefulWidget {
+  @override
+  _HomeScreenState createState() => _HomeScreenState();
+}
+
+class _HomeScreenState extends State<HomeScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      debugShowCheckedModeBanner: false,
+      home: Scaffold(
```

Comment:
@ItsAdityaKSingh you do not need to introduce a new Material App here as we need to makea central theme and routing system. You can just start with a new Scaffold.

Reply:
Done